packages/api-derive/src/chain/bestNumber.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017-2019 @polkadot/ui-react-rx authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import ApiRx from '@polkadot/api/rx';
+import { BlockNumber, Header } from '@polkadot/types/index';
+import { Observable } from 'rxjs';
+import { filter, map } from 'rxjs/operators';
+
+/**
+ * Get the latest block number.
+ */
+export function bestNumber (api: ApiRx) {
+  return (): Observable<BlockNumber> =>
```

Comment:
I went this way so that it's a pure currified function, should be easy to memoize. However this won't work that well with typedoc.